quickwit-metastore/migrations/postgresql/2021-09-02-060554_create_splits/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE splits (
+    split_id VARCHAR(50) PRIMARY KEY,
+    split_state VARCHAR(30) NOT NULL,
+    start_time_range BIGINT,
```

Comment:
I think it is ok not to have an index on start_time_ragne, end_time_range, and tags at the moment.